### PR TITLE
zendesk widget wording changes

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

File: dist/js/common-header.js [hunk 1]
```diff
@@ -7695,49 +7738,49 @@ module.run(["$templateCache", function($templateCache) {
   $templateCache.put("subscription-status-template.html",
     "<div ng-show=\"!expandedFormat\">\n" +
     "  <h3 ng-disable-right-click>\n" +
-    "    <span ng-show=\"subscriptionStatus.statusCode !== 'not-subscribed'\" ng-bind-html=\"messagesPrefix + '.' + subscriptionStatus.statusCode + subscriptionStatus.plural | translate:subscriptionStatus | to_trusted\"></span>\n" +
+    "    <span ng-show=\"subscriptionStatus.statusCode !== 'not-subscribed'\" ng-bind-html=\"'subscription-status.' + subscriptionStatus.statusCode + subscriptionStatus.plural | translate:subscriptionStatus | to_trusted\"></span>\n" +
     "  </h3>\n" +
     "  \n" +
     "  <span ng-show=\"subscriptionStatus.statusCode === 'trial-available'\">\n" +
     "    <button class=\"btn btn-primary btn-xs\" ng-click=\"showStoreModal = true;\">\n" +
-    "      <span translate=\"{{messagesPrefix}}.start-trial\"></span>\n" +
```
Comment (ezequielc) [inline dist/js/common-header.js]: @settinghead I think your changes are reverting the updates from component-subscription-status. You might need a `bower update` and `gulp build` again. Thanks

Comment (settinghead) [inline dist/js/common-header.js]: @ezequielc Good to know. Thanks. Was wondering why these unrelated changes came up.

File: dist/js/common-header.js [hunk 2]
```diff
@@ -7553,8 +7604,8 @@ angular.module("risevision.widget.common.subscription-status")
     ["risevision.common.config",
      "risevision.widget.common.subscription-status.config"])
     .service("subscriptionStatusService", ["$http", "$q", "STORE_SERVER_URL", 
-    "PATH_URL", "AUTH_PATH_URL",
-    function ($http, $q, STORE_SERVER_URL, PATH_URL, AUTH_PATH_URL) {
+    "PATH_URL", "AUTH_PATH_URL", "PATH_URL_BY_DISPLAY_ID",
```
Comment (settinghead) [inline dist/js/common-header.js]: @ezequielc I updated everything to latest version, now it looks like some of the recent changes got leaked into the build (https://github.com/Rise-Vision/component-subscription-status/commit/a3ec865db91f2d832e4a0156a645dca863eeafda). Does this look OK?

Comment (ezequielc) [inline dist/js/common-header.js]: @settinghead We just merged a new CH version. If you rebase, you should not see these changes anymore. Thanks

**Comment by settinghead** [comment]
@ezequielc Updated the code to v1.18.19 and also applied latest changes. Is the PR ready to merge? Thanks.
